hw/ip/otbn/data/otbn.hjson

```diff
@@ -382,6 +382,11 @@
                   <td>BUSY_SEC_WIPE_IMEM</td>
                   <td>OTBN is busy securely wiping the instruction memory.</td>
                 </tr>
+                <tr>
+                  <td>0x10</td>
+                  <td>INIT_SEC_WIPE</td>
+                  <td>OTBN is busy with the initial internal secure wipe.</td>
+                </tr>
```

Comment:
Thanks for this feedback, @GregAC.

I considered that at first too, but I decided to introduce a separate status state for three main reasons:

First, I think it's cleaner and more explicit / less surprising to users: `BUSY_EXECUTE` is defined as *"OTBN is busy executing software"*. We would have to redefine it to *"OTBN is busy executing software or performing its initial internal secure wipe"*. This definition would mix two states that have nothing to do with each other: the former is a reaction to the `EXECUTE` command while the latter is performed after reset without any command from the user.

Second, I think DV benefits from separating the initial secure wipe state from that of program execution. For example, as we are removing reset from some internal registers, it is imperative for security reasons that OTBN securely wipes those registers before allowing any software to execute. With separated states, this can be a simple assertion that checks if, after every reset, OTBN conducts a secure wipe before starting program execution. Related to this, there already are assertions such as
https://github.com/lowRISC/opentitan/blob/6a643647dde9851c1b612d7219e783a03713946f/hw/ip/otbn/dv/uvm/env/otbn_scoreboard.sv#L413-L415
where the condition would no longer be sufficient.

Third, I think it's more future-proof: In a follow-up PR, we may introduce a command to trigger a secure wipe of the internal state (proposed in https://github.com/lowRISC/opentitan/issues/13499). For reasons of symmetry with the `BUSY_SEC_WIPE_{D,I}MEM` states, I'd expect OTBN to enter a `BUSY_SEC_WIPE_INT` (or similar) state upon receiving that command. The current `INIT_SEC_WIPE` state could then become that state. In fact, an alternative to the current implementation would be to already name the state `BUSY_SEC_WIPE_INT`.

Reply:
My main concern is `BUSY_EXECUTE` already includes a secure wipe (namely the one happening after you've completed the OTBN operation) so the new state would mean we've got two states we use whilst waiting for a secure wipe to finish (so the current documented definition isn't strictly correct).

I agree with your second and third points about introducing a separate state. 

The key here is to immediately introduce the `BUSY_SEC_WIPE_INT` state which we'll start in after reset and transition to after `BUSY_EXECUTE` when we're doing the secure wipe at the end of an operation.

Reply:
Adding `BUSY_SEC_WIPE_INT` in #13654 and using that state in this PR.